build/resolver/git/entrypoint.sh

```diff
@@ -223,7 +223,10 @@ pull() {
         fi
     fi
 
-    cd $WORKDIR/data
+    # filter sensitive information
```

Comment:
what's the difference with this command? Can you add more annotation to explain what it changed and why it comes out?

Reply:
PTAL